app/models/feeds/apple_subscription.rb

```diff
@@ -23,6 +25,14 @@ class Feeds::AppleSubscription < Feed
   validate :must_be_private
   validate :must_have_token
 
+  # for soft delete, need a unique slug to be able to make another
+  def paranoia_destroy_attributes
+    {
+      deleted_at: current_time_from_proper_timezone,
+      slug: "#{slug} - #{Time.now.to_i}"
```

Comment:
Enabled deleting apple feeds, but to make a new one, the old one has to be deleted and have a distinct slug, or there is a postgres uniq index that will prevent inserting another apple feed